Jenkinsfile

```diff
@@ -1,3 +1,7 @@
+import org.jenkinsci.plugins.pipeline.modeldefinition.Utils
+
+DOCKER_IMAGE = 'rocm/migraphx-ci-ubuntu'
```

Comment:
This DOCKER_IMAGE value was taken from the Github Actions file. Should we change the repository since these pipelines use different Dockerfiles?

Reply:
The docker is the build environment, not the build of migraphx itself.  Essentially the prereqs.  If the commit hash (not :latest) is the same, then the docker would be valid even from that repo.  The challenge would come from the case where the PR is brand new so the github side didnt create it yet.  Then each job would be building and perhaps attempting to publish at the same time.

Reply:
That docker image is built from hip-clang.docker. Its not the same as what used on jenkins which is based on the Dockerfile.